Classes/Service/NodePropertyValidationService.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+namespace Neos\Neos\Ui\Service;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Validation\Validator\ValidatorInterface;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class NodePropertyValidationService
+{
+
+    /**
+     * @param $value
+     * @param string $validatorName
+     * @param array $validatorConfiguration
+     * @return bool
+     */
+    public function validate($value, string $validatorName, array $validatorConfiguration): bool
```

Comment:
Is this compatible with all the PHP versions that Neos 3.3 supports?

Reply:
Thx for the hint, this one is, but https://github.com/neos/neos-ui/pull/2351/files#diff-b1f15cbe77fdc0aceaae2512fd4a3c12R48 isn't due the nullable return type.